lms/djangoapps/instructor_task/api_helper.py

```diff
@@ -75,6 +95,42 @@ def _reserve_task(course_id, task_type, task_key, task_input, requester):
     return InstructorTask.create(course_id, task_type, task_key, task_input, requester)
 
 
+def generate_already_running_error_message(task_type):
+    """
+
+    Returns already running error message for given task type.
+    """
+
+    message = ''
+    # already_running_message_template = "The {report_type} report is being created. " \
+    #                                    "To view the status of the report, see Pending Tasks below. " \
+    #                                    "You will be able to download the report when it is complete."
+    report_types = {
+        'grade_problems': _('problem grade'),
+        'problem_responses_csv': _('problem responses'),
+        'profile_info_csv': _('enrolled learner profile'),
+        'may_enroll_info_csv': _('enrollment'),
+        'detailed_enrollment_report': _('detailed enrollment'),
+        'exec_summary_report': _('executive summary'),
+        'course_survey_report': _('survey'),
+        'proctored_exam_results_report': _('proctored exam results'),
+        'export_ora2_data': _('ORA data'),
+        'grade_course': _('grade'),
+
+    }
+
+    if report_types.get(task_type):
+
+        message = _("The {report_type} report is being created. "
+                    "To view the status of the report, see Pending Tasks below. "
+                    "You will be able to download the report when it is complete.").format(
```

Comment:
Nit: This may read better if the ending parenthesis is on its own line.  So:
```
message = _(
    "The {report_type} report is being created. "
    "To view the status of the report, see Pending Tasks below. "
    "You will be able to download the report when it is complete."
).format(report_type=report_types.get(task_type))
```